webpack.config.js

```diff
@@ -120,30 +120,9 @@ function webpackForTest(testFileName) {
       },
     },
   }
-  // return Object.assign({}, getDefaultConfiguration(), test)
   return test
 }
 
-function webpackIntegrationTests() {
-  const dir = './test/integration/'
-  const tests = []
-  const dirPaths = fs.readdirSync(dir)
-  tests.push(webpackForTest(`./${path.join(dir, 'integration.ts')}`))
-  const subdirs = dirPaths.filter(
-    (filename) =>
-      !filename.match(/\/?([^\/]*)\.ts$/) && filename !== 'README.md',
-  )
-  subdirs.forEach((subdir) => {
-    const subdirPaths = fs.readdirSync(path.join(dir, subdir))
-    subdirPaths.forEach((filename) => {
-      tests.push(webpackForTest(`./${path.join(dir, subdir, filename)}`))
-    })
-  })
-  return tests.map(
-    (test) => (env, argv) => Object.assign({}, getDefaultConfiguration(), test),
-  )
-}
-
```

Comment:
Same here, I think this file is desireable - Maybe you need to rebase on 2.0 again? (Not sure)